meta-zephyr-sdk/recipes-devtools/xilinx_qemu/xilinx-qemu_git.bb

```diff
@@ -0,0 +1,211 @@
+#  --enable-vhost-net       enable vhost-net acceleration support
+#  --enable-trace-backends=B Set trace backend
+#                           Available backends: $($python $source_path/scripts/tracetool.py --list-backends)
+#  --with-trace-file=NAME   Full PATH,NAME of file to store traces
+#                           Default:trace-<pid>
+#  --disable-spice          disable spice
+#  --enable-spice           enable spice
+#  --enable-rbd             enable building the rados block device (rbd)
+#  --disable-libiscsi       disable iscsi support
+#  --enable-libiscsi        enable iscsi support
+#  --disable-libnfs         disable nfs support
+#  --enable-libnfs          enable nfs support
+#  --disable-smartcard-nss  disable smartcard nss support
+#  --enable-smartcard-nss   enable smartcard nss support
+#  --disable-libusb         disable libusb (for usb passthrough)
+#  --enable-libusb          enable libusb (for usb passthrough)
+#  --disable-usb-redir      disable usb network redirection support
+#  --enable-usb-redir       enable usb network redirection support
+#  --enable-lzo             enable the support of lzo compression library
+#  --enable-snappy          enable the support of snappy compression library
+#  --disable-guest-agent    disable building of the QEMU Guest Agent
+#  --enable-guest-agent     enable building of the QEMU Guest Agent
+#  --with-vss-sdk=SDK-path  enable Windows VSS support in QEMU Guest Agent
+#  --with-win-sdk=SDK-path  path to Windows Platform SDK (to build VSS .tlb)
+#  --disable-seccomp        disable seccomp support
+#  --enable-seccomp         enables seccomp support
+#  --with-coroutine=BACKEND coroutine backend. Supported options:
+#                           gthread, ucontext, sigaltstack, windows
+#  --disable-coroutine-pool disable coroutine freelist (worse performance)
+#  --enable-coroutine-pool  enable coroutine freelist (better performance)
+#  --enable-glusterfs       enable GlusterFS backend
+#  --disable-glusterfs      disable GlusterFS backend
+#  --enable-gcov            enable test coverage analysis with gcov
+#  --gcov=GCOV              use specified gcov [$gcov_tool]
+#  --disable-tpm            disable TPM support
+#  --enable-tpm             enable TPM support
+#  --disable-libssh2        disable ssh block device support
+#  --enable-libssh2         enable ssh block device support
+#  --disable-vhdx           disables support for the Microsoft VHDX image format
+#  --enable-vhdx            enable support for the Microsoft VHDX image format
+#  --disable-quorum         disable quorum block filter support
+#  --enable-quorum          enable quorum block filter support
+#  --disable-numa           disable libnuma support
+#  --enable-numa            enable libnuma support
+
+#--disable-fdt: Required for arm-generic-fdt machine type
+
+QEMUS_BUILT = "aarch64-softmmu"
+QEMU_FLAGS = "--disable-docs  --disable-sdl --disable-debug-info  --disable-cap-ng \
```

Comment:
Should these flags mostly be in sync with the standard qemu recipe?

Noticing some minor differences (standard qemu has `--disable-libssh` and `--disable-iconv`).  Would be good to either keep in sync or comment in commit message as to why we differ.

Reply:
```
$ /opt/zephyr-sdk/2.2/sysroots/x86_64-pokysdk-linux/usr/bin/qemu-system-aarch64 --version
QEMU emulator version 4.1.0 (v4.1.0-dirty)
Copyright (c) 2003-2019 Fabrice Bellard and the QEMU Project developers

$ /opt/zephyr-sdk/2.2/sysroots/x86_64-pokysdk-linux/usr/bin/qemu-system-xilinx-aarch64 --version
QEMU emulator version 2.11.1 (v2.6.0-13294-g6617fbc8be-dirty)
Copyright (c) 2003-2017 Fabrice Bellard and the QEMU Project developers
```

We are currently checking out v2019.2 of Xilinx QEMU fork which is version 2.11.1. This QEMU version does not support `--disable-libssh` and `--disable-iconv`.

Reply:
Ah, I see.  Let's comment on that as well.

Curious if you've tried `master-next` branch.

Reply:
The `master-next` branch supports these flags (since it merges the upstream QEMU 4.1.0), but it has not been officially released by Xilinx, so I would say the latest official release by Xilinx (v2019.2) is a safer bet.